src/main/java/com/google/sps/data/algorithm/TspSolver.java

```diff
@@ -156,10 +162,8 @@ public Tuple solverHelper(int currPos, int numNodes, LocalTime currentTime,
      */
     for (int i = 0; i < numNodes; i++) {
       // set open boolean and time to add to currenTime
-      boolean isOpen = (boolean) checkIsOpenSetTime(currentTime, i,
-          currPos).get(0);
-      int timeToAdd = (int) checkIsOpenSetTime(currentTime, i,
-          currPos).get(1);
+      boolean isOpen = travelPlusWaitTime(currentTime, i, currPos) != -1;
+      int timeToAdd = travelPlusWaitTime(currentTime, i, currPos);
```

Comment:
nit: just call this method one time then check if timeToAdd is -1 afterwards. Saves the trouble of computing the value twice.

Reply:
Done!